.gitlab/ci/.gitlab-ci.global.yml

```diff
@@ -543,6 +543,27 @@
     - !reference [ .validate_content_test_conf_branch_merged ]  # This section should be the last one in the script, do not move it.
     - job-done
 
+.run-validations-new-validate-flow:
+  stage: unittests-and-validations
+  extends:
+    - .default-job-settings
+  variables:
+    KUBERNETES_CPU_REQUEST: 1000m
```

Comment:
this variable is already set in `.gitlab/ci/.gitlab-ci.variables.yml` file and is set to 2000m